### DISCUSSION TITLE
add link to self-contained html

### DISCUSSION THREAD

**Discussion by bugsbunny88** [quarto-dev/quarto-cli#3449] [Q&A] [answered]
Hey guys I have a self contained static webpage "xyz.html" that I generated in Python.
I like using Quatro but for this 1 specific page I just want to add it to the navigation bar. 
So if someone clicks on the navigation bar link, it takes them to this page . I know its outside of the Quarto ecosystem and they won't be able to go back with navigation but its a simple way for me to implement. Any help on how to do this ? 

I tried this in _quarto.yml but it didn't work...

 - section: "Basics"
          contents:
            - index.qmd
            - [Gas](gass_dash.html){:target="_blank"}
            - basics-jupyter.qmd

I have gass_dash.html in the root directory because I don't really know best practice where to put it.

**Comment by cscheid** [comment, marked as answer]
add:

```yaml
- resources:
  - xyz.html
```

to your document's front matter.

**Reply by bugsbunny88** [reply]
Sorry to just clarify ; 
for _quarto.yml ; I have
```
contents:
            - index.qmd
            - gas.qmd
```
a seperate file in root as gas.qmd which contains

```
---
title: "Test"
resources:
  - gas.html
---
```

but I end up with a blank page.. so assuming I'm doing something wrong ? can gas.html remain in root prior to doing quarto render ?

**Reply by cscheid** [reply]
your quarto-generated page and your other page can't have the same name.

**Reply by bugsbunny88** [reply]
ah gotcha perfect ; everything works beautifully now. thank you @cscheid
